src/EcsRx.Plugins.GroupBinding/Systems/Handlers/GroupBindingSystemHandler.cs

```diff
@@ -36,25 +37,26 @@ public GroupBindingSystemHandler(IObservableGroupManager observableGroupManager)
         public bool CanHandleSystem(ISystem system)
         { return true; }
 
-        public IGroup GetGroupAttributeIfAvailable(ISystem system, MemberInfo member)
+        public Tuple<IGroup, int[]> GetGroupAndAffinityFromAttributeIfAvailable(ISystem system, MemberInfo member)
```

Comment:
Could we use a named tuple here so its easier for consumer to see whats happening?
https://stackoverflow.com/questions/51670844/how-to-return-named-tuples-in-c

Reply:
I didn't want to add a reference to System.ValueTuple just for that.
I made a dedicated class and used that instead.